unity/ComputerVision-UnityClient/Assets/Scripts/NetworkManager.cs

```diff
@@ -239,7 +266,7 @@ void UpdateStats()
 
         public void UpdateAverageLatency(ConcurrentQueue<float> queue, float requestLatency)
         {
-            if (queue.Count < 10)
+            if (queue.Count < 100)
```

Comment:
Is 100 too high to see any changes in a reasonable amount of samples?

Reply:
@brucearmstrong  What is your take in that? I updated that based on the JS client & android app RollingAverageCalculation

Reply:
If I have full latency of 80 ms, then the average should change every 8 seconds

Reply:
The 100 value in RollingAverage.java is the maximum size of a rolling window of data points. On sample #101, sample #1 is dropped.